sed/loader/flash/loader.py

```diff
@@ -178,6 +177,24 @@ def available_channels(self) -> List:
         available_channels.remove("pulseId")
         return available_channels
 
+    @property
+    def schema_from_config(self) -> set:
+        """Returns the set of all channels of the dataframe based on the config file
+
+        This is the complete list of channels which will be in the dataframe after creation.
+        """
+
+        schema = {"electronId", "trainId", "pulseId"}
+        if self._config["dataframe"]["split_sector_id_from_dld_time"]:
+            schema.add(self._config["dataframe"]["sector_id_column"])
+        for channel in self.available_channels:
+            if channel == "dldAux":
+                for key in self._config["dataframe"]["channels"][channel]["dldAuxChannels"].keys():
+                    schema.add(key)
```

Comment:
Similar here. Aux channel does not have to be defined in the config.

Reply:
At flash, the aux channel is always present, might consider adding it to the defaults